src/app/components/game-components/game-component/game-component.scss

```diff
@@ -105,12 +112,18 @@
 .last-move-stroke {
     stroke: var(--last-move);
 }
+.last-move-stroke.strokeless {
+    fill: var(--last-move);
+}
 .victory-fill {
     fill: var(--victory);
 }
 .victory-stroke {
     stroke: var(--victory);
 }
+.victory-stroke.strokeless {
```

Comment:
aka victory-fill ? un stroke strokeless c'est louche quand même :D

Reply:
comment on nommerais les zones polylinesques calculée pour remplacer le strokes ?
Car c'est ça que son les strokeless actuellement :p

Reply:
ah, comment on nommerais les strokes MGP qui sont pas des strokes SVG? :D Bonne question!
hand-stroke?
manual-stroke?

sinon, autre point ici: `.victory-stroke .strokeless` c'est mieux que sans espace, `.a .b` c'est pour dire `classe a ET classe b`, plus clair avec un espace

Reply:
mgp-stroked :D ?
calculated-stroke ?
Manual est tentant, hand pas trop.
done pour .a .b

Reply:
ah non, ça casse le behavior ! j'ai testé avec et sans espace, sans l'espace, ELNEPU l'petit highlight manuel !

Reply:
HAN bizarrerie !
Du coup, j'serais plutôt partant pour `manual-stroke` in fine !

Reply:
[some timestamp] private css lord that I know: .a.b means that you target an element that has both the class a & b
[some timestamp] private css lord that I know: .a .b targets element with class b in an element with class a
APPARENTEMENT !

Reply:
OH ma recherche rapide sur les internet m'avais donc menti et on veut .a.b ! TPR